openmsupply/open-msupply
# 1185 negative number of packs part 02

Fixes #1185 

# 👩🏻‍💻 What does this PR do? 
Change StockLinesReducedBelowZero and AdjustmentReasonNotProvided into structural errors. 

# 🧪 How has/should this change been tested? 
1. Create a new outbound shipment and add an invoice line with 1 pack left
2. Create a stocktake with the stock line that was added to the invoice line and put any number into counted number of packs
3. See error.

#### Confirming stocktake
- Create a stocktake with some stock lines with less counted number of packs
- Create a new outbound shipment with the same stock lines and leave 1 packs left
- Go back to stocktake and try to finalise stocktake
